index.js

```diff
@@ -5,9 +5,13 @@ var data = [
   ["3", "Jorge", "img/nophoto.jpg", "Marketing", "3120"],
 ];
 
-new Lightsheet(document.getElementById("lightsheet"), {
-  data,
-  onCellChange: (colIndex, rowIndex, newValue) => {
-    console.log(colIndex, rowIndex, newValue);
+new Lightsheet(
+  {
+    data: data,
+    sheetName: "Sheet1",
+    onCellChange: (colIndex, rowIndex, newValue) => {
+      console.log(colIndex, rowIndex, newValue);
+    },
   },
-});
+  document.getElementById("lightsheet"),
```

Comment:
targetelement should be the first parameter. because target element is a required parameter. the whole options object should be optional